src/subdomains/supporting/support-issue/services/support-issue.service.ts

```diff
@@ -52,6 +69,9 @@ export class SupportIssueService {
       newIssue.additionalInformation = dto.transaction;
     }
 
+    // limit request
+    if (dto.limitRequest) await this.limitRequestService.increaseLimitInternal(dto.limitRequest, userData);
```

Comment:
Create the limit request with support issue? Or is this updated when removing the limit request API?

Reply:
Yes right, we create a new limitRequest when creating a limitRequestSupportIssue. I think I don't understand your question 😅 
Why should it be updated when removing the limitRequest Endpoints? The limitRequest entity/service will still be used when the endpoints are removed